_posts/2023-06-30-july-10-downtime.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Downtime July 10th, 2023
+state: active
+severity: 0
+---
+
+We will be performing upgrades to some of our hardware. Much of our
+services will be down for a large part of July 10th, 2023.
+
+Please be aware of the following outages:
+
+1. NEU Cache Servers
+1. SSO service, this means you will be unable to authenticate
+1. Kaizen HIL/BMI service
```

Comment:
We should communicate that the HIL/BMI servers will need to be powered off if they are booted using BMI because the iSCSI servers will be powered off and moved.